en/docs/try-out/account-and-transaction-flow.md

```diff
@@ -13,55 +14,68 @@ This document provides step by step instructions to deploy, subscribe, and invok
 
 3. Select **OpenAPI File/Archive**. ![create-an-api](../assets/img/get-started/quick-start-guide/create-an-api.png)
 
-4. Click **Browse File to Upload** and select the relevant API from the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources/apis` directory.
+4. Click **Browse File to Upload** and select the relevant API from the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources/apis/consumerdatastandards.org.au/1.8.0/consumer-data-standards-1.8.0.yaml` directory.
```

Comment:
Since you have given the file name of the yaml file. Let's reword this.
`and select the 
<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources/apis/consumerdatastandards.org.au/1.8.0/consumer-data-standards-1.8.0.yaml` file.

Reply:
@silvashenol20 this is not fixed yet.

Reply:
![image](https://user-images.githubusercontent.com/100564285/164649465-bab8991c-0ddc-4427-8af3-b4898529a836.png)

Reply:
noted